integration-test/src/python/test_runner/main.py

```diff
@@ -14,18 +14,18 @@
 # The location of default configure file
 DEFAULT_TEST_CONF_FILE = "integration-test/src/python/test_runner/resources/test.json"
 
-RETRY_ATTEMPTS = 15
+RETRY_ATTEMPTS = 25
```

Comment:
When there are legit failures retrying 25 times is excessive and it takes over 4 minutes of results checking per test. Even 15 times seems high. If we have to retry anything this many times for simple, small test topologies, something is wrong.

That said, I can keep it if our tests are known to be too flaky when set below 25.